src/game/common/system/gamememoryinit.cpp

```diff
@@ -648,6 +648,7 @@ static PoolSizeRec UserMemoryPools[] = {
     { "ThumbnailManagerClass", 32, 32 },
     { "SmudgeSet", 32, 32 },
     { "Smudge", 128, 32 },
+    { "Win32BufferedFile", 32, 32 }, // Thyme specific.
```

Comment:
Can we call this something other than Win32BufferedFile. I know the original implementation is called Win32File and I only left it called that because that is what it was originally called despite it using the posix API. StandardBufferedFile or even just StandardFile would be okay given that its using stdio.h functions.

Reply:
I have chosen `Win32BufferedFile` so it is very obvious that it is similar to `Win32LocalFile`. I can change name if you really desire so.

Reply:
Renamed to `StandardFile`